python/lsst/meas/algorithms/defects.py

```diff
@@ -59,26 +60,44 @@ class Defects(collections.abc.MutableSequence):
     defectList : iterable of `lsst.meas.algorithms.Defect`
                  or `lsst.geom.BoxI`, optional
         Collections of defects to apply to the image.
+    metadata : `lsst.daf.base.PropertyList`, optional
+        Metadata to associate with the defects.  Will be copied and
+        overwrite existing metadata.  If not supplied the existing
+        metadata will be reset.
```

Comment:
I wrote (well, copied and pasted) this comment, but I'm not sure it makes sense — if I am constructing a new `Defects`, what existing metadata would be replaced?

Reply:
Should I add "if any" to "Will be copied and overwrite existing metadata.", in case that we have a new `Defects`, as you say?

Reply:
What “any” could there be if you're creating a new object? There is nothing that could be over-written.